Cargo.toml

```diff
@@ -59,13 +65,21 @@ lemmy_routes = { version = "=0.17.1", path = "./crates/routes" }
 lemmy_db_views = { version = "=0.17.1", path = "./crates/db_views" }
 lemmy_db_views_actor = { version = "=0.17.1", path = "./crates/db_views_actor" }
 lemmy_db_views_moderator = { version = "=0.17.1", path = "./crates/db_views_moderator" }
-activitypub_federation = { version = "0.4.1", default-features = false, features = ["actix-web"] }
+activitypub_federation = { git = "http://github.com/cetra3/activitypub-federation-rust.git", branch = "remove_actix_rt", default-features = false, features = [
+    "actix-web",
+] }
+# activitypub_federation = { version = "0.4.1", default-features = false, features = ["actix-web"] }
```

Comment:
I published 0.4.2 with your PR now. Btw whats with the formatting changes? Would be nice to leave it as is.

Reply:
Apologies, fixed